server_cmds.go

```diff
@@ -219,7 +239,7 @@ func (s *ServerMethods) InstanceInfo() (*Instance, error) {
 // ServerConnectionInfo returns detailed bandwidth and transfer information about the selected instance.
 func (s *ServerMethods) ServerConnectionInfo() (*ServerConnectionInfo, error) {
 	r := &ServerConnectionInfo{}
-	if _, err := s.ExecCmd(NewCmd("serverrequestconnectioninfo").WithResponse(&r)); err != nil {
+	if _, err := s.ExecCmd(NewCmd("hostinfo").WithResponse(&r)); err != nil {
```

Comment:
this is now using the wrong method.

Reply:
Yep I noticed this today. I am sorry I got that mixed up. I corrected that in my latest commit.